modules/core/shared/src/test/scala/gem/arb/ArbCoAdds.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2016-2018 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package arb
+
+import gem.syntax.prism._
+
+import org.scalacheck.{ Arbitrary, Cogen, Gen }
+
+trait ArbCoAdds {
+
+  implicit val arbCoAdds: Arbitrary[CoAdds] =
```

Comment:
Should `CoAdds` be restricted to only positive numbers in the type?

Reply:
Ideally yes.  We have a number of types like this but we haven't settled on a better approach.  At any rate here I'm just moving this code out of `gem.Arbitraries` into an individually importable module like we've done for a number of other types.